src/BenchmarkDotNet.Analyzers/BenchmarkRunner/RunAnalyzer.cs

```diff
@@ -54,14 +54,14 @@ public class RunAnalyzer : DiagnosticAnalyzer
         description: AnalyzerHelper.GetResourceString(nameof(BenchmarkDotNetAnalyzerResources.BenchmarkRunner_Run_GenericTypeArgumentClassMustBeAnnotatedWithAGenericTypeArgumentsAttribute_Description)));
 
 
-    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
-    [
+    public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => new DiagnosticDescriptor[]
+    {
         TypeArgumentClassMissingBenchmarkMethodsRule,
         TypeArgumentClassMustBePublicRule,
         TypeArgumentClassMustBeUnsealedRule,
         TypeArgumentClassMustBeNonAbstractRule,
         GenericTypeArgumentClassMustBeAnnotatedWithAGenericTypeArgumentsAttributeRule,
-    ];
+    }.ToImmutableArray();

```

Comment:
Is there any reason why you can't use collection expressions here?

Reply:
v2.8 uses an older version of immutable collections package that doesn't support it. I tried manually adding the latest version, but the compiler complained that it couldn't find the assembly when it ran the analyzer.